### PR TITLE
Adding KubeProxyConfigMap19 checking in test-e2e-kubeadm

### PR DESCRIPTION
**Pull request by lubinsz**
Signed-off-by: Bin Lu <bin.lu@arm.com>

**What type of PR is this?**
/kind testing

**What this PR does / why we need it**:
Adding KubeProxyConfigMap19 checking in test/e2e_kubeadm/proxy_addon_test.go

**Which issue(s) this PR fixes**:
None.

**Special notes for your reviewer**:
None.
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e_kubeadm/proxy_addon_test.go [hunk 1]
```diff
@@ -75,6 +77,22 @@ var _ = Describe("proxy addon", func() {
...
+			gomega.Expect(m).To(gomega.HaveKey("kubernetesVersion"))
+			k8sVersionString := m["kubernetesVersion"].(string)
+			k8sVersion, err := version.ParseSemantic(k8sVersionString)
+			if err != nil {
+				framework.Failf("error reading kubernetesVersion from %s ConfigMap: %v", kubeadmConfigName, err)
+			}
+
+			if k8sVersion.Major() >= 1 && k8sVersion.Minor() >= 9 {
+				gomega.Expect(cm.Data).To(gomega.HaveKey(kubeProxyConfigMapKey19))
+			}
```
Comment (neolit123) [inline test/e2e_kubeadm/proxy_addon_test.go]: @lubinsz 
please explain the motivation for adding this test?

/cc @fabriziopandini

Comment (neolit123) [reply test/e2e_kubeadm/proxy_addon_test.go]: should this be called `kubeProxyConfigMapKeyKubeconfig`?

Comment (lubinszARM) [reply test/e2e_kubeadm/proxy_addon_test.go]: > @lubinsz
> please explain the motivation for adding this test?
> 
> /cc @fabriziopandini

KubeProxyConfigMap19 is created by kubeadm, and used for Kubernetes 1.9 and above.
It is defined in kubeadm/app/phases/addons/proxy/manifests.go.
And this configmap is installed in cmd/kubeadm/app/cmd/phases/init/addons.go
I think we can check it in kubeadm related test cases.

Comment (lubinszARM) [reply test/e2e_kubeadm/proxy_addon_test.go]: > should this be called `kubeProxyConfigMapKeyKubeconfig`?

OK.

Comment (neolit123) [reply test/e2e_kubeadm/proxy_addon_test.go]: the name of the constants in kubeadm/app/phases/addons/proxy/manifests.go is not important and can be changed. better the constant in this PR without the "19".

your test is adding the verification for the existence of the kubeconfig.conf key in the kube-proxy ConfigMap:
https://github.com/kubernetes/kubernetes/blob/5b47a92ad8668612d912ced2f0e6c658b9325727/cmd/kubeadm/app/phases/addons/proxy/manifests.go#L30

and the scope of the PR is a bit misleading.
inspecting the contents of the config map is OK, but this should not be bound to a version check as 1.9 is long out of support.

Comment (lubinszARM) [reply test/e2e_kubeadm/proxy_addon_test.go]: @neolit123 
Done.

**Review by fabriziopandini — APPROVED** [review]
@lubinsz thanks for this PR!
I have to admit that the 19 things in the title tricked me a little bit (1.9 is ages ago...), but after some code archeology I am definitely +1 on adding this test
/approve

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Pull request merged**